client/app/screens/LoginScreen.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { AppRegistry, StyleSheet, Text, View, TextInput, ScrollView, } from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
```

Comment:
Icons are actually included in native-base as well, https://docs.nativebase.io/COMPONENTS.html#Icon_Default. Sorry if I sound like a broken record 😆